argon2/include/blake2b.h

```diff
@@ -0,0 +1,94 @@
+#ifdef __cplusplus
```

Comment:
Should we make this change as we integrate so we can make use of the most up-to-date blake2?